src/aatoolbox/datasources/ecmwf_api/ecmwf_api_download_seasonal_monthly.py

```diff
@@ -42,14 +45,16 @@
 PRATE_DIR = "prate"
 
 
-def download_date(
+def download(
     # question: do we want to assume an iso3 here?
     # might want to e.g. download for a multi-country region or only a city
     iso3: str,
-    date_forec: Union[str, date],
     ecmwf_dir: Union[str, Path],
     iso3_gdf: gpd.GeoDataFrame,
+    date_list: List[Union[str, date]] = None,
     area: Area = None,
+    clobber: bool = False,
+    grid: float = 0.4,
```

Comment:
I should have added this in when I did it, but if we want to allow the user to vary the grid, it should maybe be included in the filename